app/services/course/assessment/question/programming/cpp/cpp_package_service.rb

```diff
@@ -0,0 +1,248 @@
+# frozen_string_literal: true
+class Course::Assessment::Question::Programming::Cpp::CppPackageService < \
```

Comment:
I was thinking of waiting till I got the Java package done, since the Java package will have one on its own. Then extract the similarities and let them inherit from `Course::Assessment::Question::Programming::LanguagePackageService`. Then all the refactoring will be done in a separate PR after the Java package is complete. What do you think?